client/src/main/java/gui/playing/PlayBoardController.java

```diff
@@ -219,5 +227,13 @@ private void putFieldOnOpponentBoard(Field field) {
   public void setMessageProcessor(MessageProcessor messageProcessor) {
     this.processor = messageProcessor;
   }
+
+  private void suspend() {
+    try {
+      Thread.sleep(1500);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
use logger and catch InterruptedEcception; as sonar siad: Thread.currentThread().interrupt();
Calling Thread.currentThread().interrupt() sets the interrupted flag (again), which is useful indeed if we want to ensure that the interrupt gets noticed and processed on a higher level.